js/tabs.js

```diff
@@ -59,11 +59,13 @@ $(document).ready(function () {
 });
 
 function getOS() {
-  var ua = navigator.userAgent.toLowerCase();
-  if (ua.indexOf("win") !== -1)
+  var ua = navigator.userAgent;
+  if (ua.indexOf("Win") !== -1)
     return "windows";
-  if (ua.indexOf("mac") !== -1)
+  if (ua.indexOf("Mac") !== -1)
     return "macos";
-  if (ua.indexOf("linux") !== -1)
+  if (ua.indexOf("Linux") !== -1)
```

Comment:
You could combine these two into one expression, since they return the same thing.

Reply:
I did think of that and wondered if it was better this way in case it changes in future; however, that seems unlikely, so they're merged now.